### ISSUE TITLE
"Markane is not installed" on VSCodium

### ISSUE THREAD

**Issue by colerehbein** [garlicbreadcleric/vscode-markane#1]
When trying to install/initialize the extension on VSCodium, it returns an error saying "Markane is not installed" despite showing as installed in the Extensions view. Problem persists after reloading VSCodium and reinstalling the extension.

**Comment by garlicbreadcleric** [comment]
@colerehbein Sorry for unintuitive error message. Markane here refers to the [language server](https://github.com/garlicbreadcleric/markane), which has to be installed separately from the extension. You can install Markane via NPM: `npm i -g markane`, for more info see [Getting started](https://github.com/garlicbreadcleric/markane/blob/main/docs/getting-started.md) docs page.

**Comment by colerehbein** [comment]
My bad for not thoroughly reading through the documentation and thank you for your quick reply, I've resolved the error. Thanks also for your work on this extension! I'll be trying it out over the next few days and am excited to see how it can replace some of what I do in Obsidian. Also grateful for your support for native markdown links versus wikilinks—foam, memo, and others don't yet support []() for links, this is the only extension I found that supports preview/etc on native links. I'm currently learning java/typescript so hopefully I can find a way to contribute to this project or similar projects in the future.
